### PR TITLE
Created login test

### PR DESCRIPTION
**Pull request by rbillings**
Addresses Issue #5

### CODE REVIEW

File: pages/resultset.py [hunk 1]
```diff
@@ -7,13 +7,15 @@
...
+from pages.base import Base
 from pages.page import Page
 
 
-class ResultsetPage(Page):
+class ResultsetPage(Base):
 
     _job_details_actionbar_locator = (By.ID, 'job-details-actionbar')
     _job_result_status_locator = (By.CSS_SELECTOR, '#result-status-pane > div:nth-child(1) > span')
+    _login_locator = (By.CSS_SELECTOR, 'a.btn > span:nth-child(1)')
```
Comment (davehunt) [inline pages/resultset.py]: This can be removed, as it is set in the parent class.

File: pages/base.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    def click_login(self):
+        self.selenium.find_element(*self._login_locator).click()
+
+    def login(self, user):
+        self.click_login()
+        browser_id = BIDPOM(self.selenium, self.timeout)
+        browser_id.sign_in(user['email'], user['password'])
+        Wait(self.selenium, self.timeout).until(
+            EC.visibility_of_element_located(self._logout_menu_item_locator))
+        return not self.is_element_visible(self._login_locator)
```
Comment (davehunt) [inline pages/base.py]: This method doesn't need to return anything, so this line can be removed.

**Comment by davehunt** [comment]
r+ wc

**Comment by rbillings** [comment]
Merging now that Travis has passed

**Pull request merged**